tsconfig.json

```diff
@@ -1,6 +1,6 @@
 {
     "compilerOptions": {
-        "outDir": "dist",
+        "outDir": "public/dist",
```

Comment:
I don't think this is correct. The public directory is for public/static content. This is meant for frontend assets. The only .ts file that should go to public is `browser.ts` because it is frontend code.

Reply:
Hmmm... ok.  That makes perfect sense.  I think I am misunderstanding how `npx` is being used to run http-server as stated in the contributing file.  Does it assume that developers have npx installed globally?  Also, it looks like that package has a lot of security vulnerabilities (when running npm audit), but maybe I am mistaken there.

Reply:
> Does it assume that developers have npx installed globally?

Yes

When you install Node.js, it also installs `npm` and `npx`. Even if you run Node.js in Docker, the official image ships with these tools.

> Also, it looks like that package has a lot of security vulnerabilities

Which package? I don't see vulnerabilities.

Reply:
I had installed it manually before you replied.  I had no idea it shipped with npm.  The more you know.